compiler-core/src/ast.rs

```diff
@@ -739,6 +740,17 @@ impl<T> Import<T> {
         }
     }
 
+    pub(crate) fn name_location(&self) -> SrcSpan {
```

Comment:
This isn't the location of the module name, that would always be the value of the `module_location` field. Perhaps "used_name_location"? Please document what this function does also